QuizNet.BusinessLogic/QuestionService.cs

```diff
@@ -47,6 +47,7 @@ public void Add(QuestionDto questionDto)
         {
             var question = _mapper.Map<Question>(questionDto);
             _questionRepository.Add(question);
+            questionDto.Id = question.Id;
```

Comment:
Nie podoba mi się to rozwiązanie. Co powiesz na zmienienie metody Add na zwracanie nowo utworzonego Id? Albo na zwracanie całego nowo utworzonego obiektu? @KrzysztofMilewski Co myślisz?

Reply:
Jasne, to ma więcej sensu, więcej wiadomo wtedy z poziomu użytkowania serwisu